src/pages/components/typography.mdx

```diff
@@ -2,10 +2,159 @@
+import CodeCode from '!!raw-loader!../../examples/typography/Code.tsx';
+
+<CodeExample code={CodeCode}>
+  <Code />
+</CodeExample>
+
+### Ellipsis text
+
+Use Ellipsis to automatically provide text content with a native title attribute and text-overflow
+styling.
+
+import Ellipsis from '../../examples/typography/Ellipsis.tsx';
+import EllipsisCode from '!!raw-loader!../../examples/typography/Ellipsis.tsx';
+
+<CodeExample code={EllipsisCode}>
+  <Ellipsis />
+</CodeExample>
+
+### Lists
+
+Use Lists to style and format ordered (`<ol>`) and unordered (`<ul>`) lists. List components come
+in small, medium, and large.
+
+import Lists from '../../examples/typography/Lists.tsx';
+import ListsCode from '!!raw-loader!../../examples/typography/Lists.tsx';
+
+<CodeExample code={ListsCode}>
+  <Lists />
+</CodeExample>
+
+### Paragraph text
+
+Use Paragraph to render `<p>` elements with the appropriate Garden styling.
+
+import Paragraph from '../../examples/typography/Paragraph.tsx';
+import ParagraphCode from '!!raw-loader!../../examples/typography/Paragraph.tsx';
+
+<CodeExample code={ParagraphCode}>
+  <Paragraph />
+</CodeExample>
+
+### Span text
+
+Use Span to style and format `<span>` elements. You can apply bold and monospace to Span.
+
+import Span from '../../examples/typography/Span.tsx';
+import SpanCode from '!!raw-loader!../../examples/typography/Span.tsx';
+
+<CodeExample code={SpanCode}>
+  <Span />
+</CodeExample>
+
+### Type modifiers
+
+You can apply two modifiers to type: bold and monospace.
+
+import Modifiers from '../../examples/typography/Modifiers.tsx';
+import ModifiersCode from '!!raw-loader!../../examples/typography/Modifiers.tsx';
+
+<CodeExample code={ModifiersCode}>
+  <Modifiers />
+</CodeExample>
+
+### Type scale
+
+Type components come in small, medium, large, extra-large, extra-extra-large and
+extra-extra-extra-large.
+
+import TypeScale from '../../examples/typography/TypeScale.tsx';
+import TypeScaleCode from '!!raw-loader!../../examples/typography/TypeScale.tsx';
+
+<CodeExample code={TypeScaleCode}>
+  <TypeScale />
+</CodeExample>
+
+## Configuration
+
+<Configuration reactPackage={props.data.mdx.package} components={props.data.mdx.components} />
+
+## API
+
+### Code
+
+<PropSheet components={props.data.mdx.components} componentName="code" />
+
+### Ellipsis
+
+<PropSheet components={props.data.mdx.components} componentName="ellipsis" />
+
+### LG
+
+<PropSheet components={props.data.mdx.components} componentName="lg" />
+
+### Ordered.List
+
+<PropSheet components={props.data.mdx.components} componentName="orderedlist" />
+
+### MD
+
+<PropSheet components={props.data.mdx.components} componentName="md" />
+
+### Paragraph
+
+<PropSheet components={props.data.mdx.components} componentName="paragraph" />
+
+### SM
+
+<PropSheet components={props.data.mdx.components} componentName="sm" />
+
+### Span
+
+<PropSheet components={props.data.mdx.components} componentName="span" />
+
+### Unordered.List
```

Comment:
Please restore this to the component name `UnorderedList`. Revisit comment: https://github.com/zendeskgarden/website/pull/100#discussion_r451086268